.github/workflows/comment-command.yml

```diff
@@ -73,9 +73,12 @@ jobs:
               content: "rocket",
             });
       - uses: actions/checkout@v4
-        with:
-          fetch-depth: 0
-          token: ${{ secrets.COMMENT_COMMAND_PAT_TOKEN }}
```

Comment:
Out of curiosity, why was this removed? The `format` job still has this.

Reply:
It's not necessary for the `pregen` job to checkout all the entire history of the repo. I added back the token though, since it needs to be able to push commits.

Reply:
> Out of curiosity, why was this removed? The `format` job still has this.

The format job needs this for wpiformats git history comparison iirc